NAS2D/Mixer/Mixer.h

```diff
@@ -19,18 +19,17 @@ namespace NAS2D
 	class Sound;
 	class Music;
 
-
 	/**
-	 * Mixer base class.
-	 *
-	 * Provides a standard Mixer interface. The base Mixer can be used
-	 * but will act as a NULL interface.
-	 */
+	 * \brief Mixer base class.
+	 * 
```

Comment:
Ahh, and we tend to periodically make style changes such as removing line trailing spaces, such as this one.